Ghidra/Processors/V850E2M/data/languages/Instructions/Branch.sinc

```diff
@@ -0,0 +1,59 @@
+#####################################################
+#####		       Branch			#####
+#####################################################
+
+
+#Bcond adr9 - ddddd1011dddcccc
+:b^c0003 adr9 is op0710=0xB & c0003 & adr9
+{
+	if (c0003) 
+		goto adr9;
+}
+:br adr9 is op0710=0xB & op0003=0x5 & adr9
+{
+	goto adr9;
+}
+
+# JARL disp22, reg2 - rrrrr11110dddddd|ddddddddddddddd0
+:jarl adr22, r1115 is (op0610=0x1E & r1115) ... & adr22
+{
+	r1115 = inst_next;
+	PC = & adr22;
+	call adr22;
+}
+
+# JARL disp32, reg1 - 00000010111RRRRR|ddddddddddddddd0|DDDDDDDDDDDDDDDD
+:jarl adr32, r0004 is op0515=0x017 & r0004; adr32
+{
+	r0004 = inst_next;
+	PC = & adr32;
+	call adr32;
+}
+
+# JMP [reg1] - 00000000011RRRRR
+:jmp [R0004] is op0515=0x003 & R0004 & op0004=0x1F
```

Comment:
There is should be second `jmp [reg1]` table, without op0004=0x1F condition. It is used for GOTOs, specifed in any register.